src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOSupport.scala

```diff
@@ -52,8 +52,10 @@ trait ElasticSearchDAOSupport extends LazyLogging {
 
   def detailFromAttribute(label: String, detail: AttributeDetail) = {
     detail match {
-      case AttributeDetail("array", Some(items)) => label -> ESDetail(items.`type`)
-      case AttributeDetail(t, None) => label -> ESDetail(detail.`type`)
+      case AttributeDetail("array", Some(items), Some(true)) => label -> new ESAggregatableType(items.`type`)
+      case AttributeDetail("array", Some(items), _) => label -> ESType(items.`type`)
+      case AttributeDetail(t, None, Some(true)) => label -> new ESAggregatableType(t)
+      case AttributeDetail(t, None, None) => label -> ESType(t)
```

Comment:
Adding in my slack comments about this block. As the cases expand, this matching will become unwieldy. If we add a new type of `vector` or `set`, for example, the cases grow exponentially. 

Possible idea - make constructors for `ESDetail` that take an `AttributeDetail` and have logic inside `ESDetail` that understands aggregatable?